bot/resources/tags/json.md

```diff
@@ -0,0 +1,15 @@
+JSON is a very handy way of storing values. But when starting out with JSON, you might run into the following error.
+
+`JSONDecodeError: Expecting value: line 1 column 1 (char 0)`
+
+What this error means is that the decoder (the reading of the JSON file) is expecting a character on line 1 column 1. In other words, your JSON is missing a critical character at the start. So it's saying that your JSON is invalid.
+Looking at how JSON is set up, you can see that they require the following at minimum if you want to start out.
```

Comment:
> your JSON is missing a critical character at the start

> they require the following at minimum if you want to start out.

This isn't correct. The JSON standard basically expects the root element to be a "value". That term has a specific definition and that's what the error is referring to. The grammar defines a "value" as any of the following:
```
object
array
string
number
"true"
"false"
"null"
```

I don't think this tag necessarily has to go into such detail. It could just point out the common cases of using an array or object as the root element. However, it should be phrased in a way that doesn't imply these are the only options.

Reply:
Thank you for this list. I have pushed a new change hopefully resolving these issues you pointed out.